src/utilities/network/lookupApi.js

```diff
@@ -22,88 +23,22 @@ const SEPARATORS = {
   POLICY_NUMBER: '-',
 };
 
-export const createPatientRecord = response => {
-  const { 
-    id,
-    name,
-    code,
-    phoneNumber,
-    billAddress1,
-    billAddress2,
-    billAddress3,
-    billAddress4,
-    billPostalZipCode,
-    emailAddress,
-    type,
-    isCustomer,
-    isSupplier,
-    isManufacturer,
-    supplyingStoreId,
-    thisStoresPatient,
-    isPatient,
-    firstName,
-    lastName,
-    dateOfBirth,
-  } = response;
-  const billingAddress = getOrCreateAddress(
-    UIDatabase,
-    billAddress1,
-    billAddress2,
-    billAddress3,
-    billAddress4,
-    billPostalZipCode
-  );
-  const patient = {
-    id,
-    name,
-    code,
-    phoneNumber,
-    billingAddress,
-    emailAddress,
-    type,
-    isCustomer,
-    isSupplier,
-    isManufacturer,
-    supplyingStoreId,
-    thisStoresPatient,
-    isPatient,
-    firstName,
-    lastName,
-    dateOfBirth,
-    isVisible: true,
-  };
-  UIDatabase.write(() => UIDatabase.update('Name', patient));
+export const createPatientRecord = patient => {
+  UIDatabase.write(() => createRecord(UIDatabase, 'Patient', patient));
+  patient?.policies?.forEach(createPolicyRecord);
 };
 
-export const createPrescriberRecord = response => {
-  const {
-    id,
-    firstName,
-    lastName,
-    registrationCode,
-    address1,
-    address2,
-    phoneNumber,
-    mobileNumber,
-    emailAddress,
-    fromThisStore,
-  } = response;
-  const address = getOrCreateAddress(UIDatabase, address1, address2);
-  const prescriber = {
-    id,
-    firstName,
-    lastName,
-    registrationCode,
-    address,
-    isVisible: true,
-    isActive: true,
-    phoneNumber,
-    mobileNumber,
-    emailAddress,
-    fromThisStore,
-  };
-  UIDatabase.write(() => UIDatabase.update('Prescriber', prescriber));
-}
+export const createPrescriberRecord = prescriber =>
+  UIDatabase.write(() => createRecord(UIDatabase, 'Prescriber', prescriber));
```

Comment:
this and the patient one seem a bit over the top

Reply:
Yeah, as above, just waiting to move to thunks!